.github/workflows/code-rabbit.yaml

```
@@ -27,15 +27,31 @@
             echo "Error: OPENAI_API_KEY is not set"
             exit 1
           fi
-      - uses: coderabbitai/ai-pr-reviewer@latest
+
+      - name: Code Review Step with Retry
+        id: code-review
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2034:warning:6:3: gh_output appears unused. Verify use (or export if used externally)

(shellcheck)